checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -39,6 +39,8 @@ def analyze_line(
             line_number: int = 0,
             context: Optional[CodeSnippet] = None,
             raw_context: Optional[CodeSnippet] = None,
+            is_added: Optional[bool] = None,
+            is_removed: Optional[bool] = None,
```

Comment:
just thinking, if a default of `False` would be better, same for the other one 🤔 any specific reason to make `None` the default?

Reply:
no, will change to False